security_considerations.md

```diff
@@ -117,3 +117,16 @@ Note also that, if used, Time Server costs are typically one signature verificat
 ### Using encrypted images on the Image repository
 
 Images stored on the Image repository may have previously been encrypted or not, at the discretion of the implementer.  The Standard does not explicitly mention using encrypted images on the Image repository because Uptane treats these blobs exactly the same as unencrypted blobs. It only imposes special requirements on images that are per-ECU encrypted on the Director repository. Therefore, there is no reason that encrypted images cannot be on the Image repository should an implementer wish to use them.
+
+### Avoiding Director replay attacks
+
+Uptane doesn't explicitly require that targets metadata from the Director contain any images at all.
+
+As such, it may be tempting to only include software images that need to be updated in Director's targets metadata, for efficiency and bandwidth-minimization reasons. However, that implies that empty targets metadata would be sent if no updates are available, and sending empty targets metadata with only the required fields present does represent a potential security threat: since empty metadata does not include any ECU- or vehicle-specific information, it could potentially be replayed to another vehicle. This would constitute a "freeze" attack, since the targeted vehicle would continue to believe it was fully updated. (See [uptane-standard issue #202](https://github.com/uptane/uptane-standard/issues/202) for a more detailed discussion.)
```

Comment:
```suggestion
As such, it may be tempting to only include software images that need to be updated in Director's Targets metadata, for efficiency and bandwidth-minimization reasons. However, that implies that empty Targets metadata would be sent if no updates are available, and sending empty Targets metadata with only the required fields present does represent a potential security threat: since empty metadata does not include any ECU- or vehicle-specific information, it could potentially be replayed to another vehicle. This would constitute a "freeze" attack, since the targeted vehicle would continue to believe it was fully updated. (See [uptane-standard issue #202](https://github.com/uptane/uptane-standard/issues/202) for a more detailed discussion.)
```